### PR TITLE
Arm: Add unit tests for xGetSAD functions

### PR DESCRIPTION
**Pull request by iragimov-arm**
Extend unit tests for xGetSAD implementations with all possible block sizes.

### CODE REVIEW

File: test/vvenc_unit_test/vvenc_unit_test.cpp [hunk 1]
```diff
@@ -1453,6 +1459,11 @@ static bool test_RdCost()
       }
     }
   }
+
+  // Additional height corner cases.
+  passed = check_SAD( &ref, &opt, num_cases, 8, 48, DF_SAD ) && passed;
+  passed = check_SAD( &ref, &opt, num_cases, 64, 24, DF_SAD ) && passed;
+  passed = check_SAD( &ref, &opt, num_cases, 128, 12, DF_SAD ) && passed;
```
[the remark this answers is not in the document]
Comment (iragimov-arm) [reply test/vvenc_unit_test/vvenc_unit_test.cpp]: Thank you for your comment!
No, I haven’t observed such sizes experimentally. When running encoding on real video files, I’ve only seen dimensions that are powers of two. The goal here was to add additional tests to support future development, so I tried to cover a wide range of scenarios. However, I found it difficult to identify clear constraints for the height.
I will remove the third test. Should the other two remain?

[the remark this answers is not in the document]
Comment (iragimov-arm) [reply test/vvenc_unit_test/vvenc_unit_test.cpp]: Great!
Thank you. Will update the patch.

**Comment by adamjw24** [comment]
I now had another look at this, and it seems that with all the strip-down that we did, the MR now is obsolete, as it does not really add any new functionality.
